falcon/response.py

```diff
@@ -202,6 +204,10 @@ def data(self):
     def data(self, value):
         self._data = value
 
+    @property
+    def headers(self):
+        return dict(self._headers)
```

Comment:
Most probably I was unclear in my comment. But I meant I had found ``.copy()`` to be the fastest.

Reply:
Just misread. Apologies. This is fixed.